src/app/lib/README.md

```diff
@@ -41,3 +41,92 @@ Here are the font sizes in `px`s:
+
+Below 1008px we define all columns to be dynamic. 
+
+Above 1007px we statically define the widths of the columns due to the fact they are limited in the UX definition, which then allows us to have a dynamic column either side of the UX grid. 
+  EG: In CSS grid values we can specific `grid-template-columns: 1fr repeat(10, minmax(0, ${staticColWidth})) 1fr;` where the `1fr`s are dymanic and the columns and `repeat()` are the static width columns.
+
+The combination of the UX grid having margins prior to the column definitions and the product requirements for a "full bleed" component creates an interesting problem. We can't use CSS paddings or margins before the CSS grid implementation otherwise all components in the grid will be limited to the padding/margin and not able to go "full bleed". Our solution to this problem stems from the implementation of the static columns above 1007px - we can define a CSS grid that has an additional column either side of the UX defined columns so that the grid-gaps act both inbetween the columns as gutters and also outside of the grid as margins. 
+**NB: This implementation means at every viewport we have an additional column either side of the UX defined columns, often as a 0px column.**
+
+Therefore to meet all requirements we implement the following CSS grid where gutters and margins are grid-gaps:
+
+| Breakpoints     | Columns           | Gutters   | Margins          | Figure |
+| :-------------- | :---------------- | :-------- | :--------------- | :----- |
+| 0px - 399px     | 6 + 2 of 0px      | 0 of 8px  | 2 of 8px         | 1      |
+| 400px - 599px   | 6 + 2 of 8px*     | 5 of 8px  | 2 of 16px        | 2      |
+| 600px - 1007px  | 6 + 2 of 0px      | 5 of 16px | 2 of 16px        | 3      |
+| 1008px - 1280px | 8 + 2 of 1fr      | 7 of 16px | 2 of 16px        | 4      |
+| 1280px +        | 10 + 2 of 1fr     | 9 of 16px | 2 of 16px        | 5      |
+
+`6 + 2 of 0px` - is based on having a 0px column either side of UX  grid so we can use CSS grid-gaps as the UX margin while allowing a grid item to be able to go "full bleed". 
+`6 + 2 of 8px*` - between 400px and 599px we have an 8px column either side of the UX grid so that the grid-gap (8px) and additional column (8px) combined meet the 16px UX margin.
+`0 of 8px` - under 400px we have only 1 column and therefore no gutters.
+
+## TL;DR 
+- We work to a UX grid
+- UX margins are exclusive of the UX column definition
+- We have a requirement to allow some items to break out of the grid
+- To achieve these requirement we've used CSS grid to define an additional column either side of the UX grid
+- 
+- This [Codepen](https://codepen.io/phil-lee/full/MPMrzJ/) is a working demo
+
+## Additional content
+
+* [Investigation PR](https://github.com/BBC-News/simorgh/pull/824)
+* [Statically defined columns for above 1280px example](https://codepen.io/phil-lee/full/zmjaMO/)
+
+### Figure 1 - Article grid layout at 300px
```

Comment:
This image is a bit misleading and doesnt show the 6 columns at the smaller widths, unlike Figure 2 which also has 6 columns but shows them, image implies <399 only has 1 column

Reply:
Good spot, this is a hangover from a definition UX column breakpoint definition - will update